### PR TITLE
Added Expenses Management UI 

### PR DESCRIPTION
**Pull request by Charul00**
issue no: #2

### CODE REVIEW

**Comment by Charul00** [comment]
@ChandelAnish review it

File: client/src/components/ExpenseManagement.css
Comment (ChandelAnish) [inline client/src/components/ExpenseManagement.css:1]: i would recommend using bootstrap instead of using vanilla css

**Review by ChandelAnish — COMMENTED** [review]
no this is not we are looking for , actually we want a expense management system which manages the net amount between the friends and all friends and add the expense on whoever the like and the system should find the net amount each person will get or has to pay to other . it's more like a splitwise app

image

**Comment by Charul00** [comment]
o got it

File: client/src/components/ExpenseManagement.css
Comment (Charul00) [reply client/src/components/ExpenseManagement.css:1]: ok

**Comment by Charul00** [comment]
@ChandelAnish check now i make simple aesthetic and loos little bit professional is this ok

**Comment by Charul00** [comment]
can you please merge it

**Review by ChandelAnish — COMMENTED** [review]
you need to update the package.json file too as react-bootstrap package is used but declared in the package.josn

**Comment by CogitoKunal** [comment]
Your Branch is Conflicting

**Pull request closed without merging**